lib/rules/landmark-main-is-top-level.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "landmark-main-is-top-level",
+  "selector": "main,[role=main]",
+  "tags": [
+	"best-practice"
+  ],
+  "metadata": {
+    "description": "Each main landmark in a document must not be nested in another landmark",
+    "help": ""
```

Comment:
The help text also goes in the aXe extension sidebar where we list out the violations.